tests/cisco_asa/show_vpn-sessiondb_anyconnect/cisco_asa_show_vpn-sessiondb_anyconnect.raw

```diff
@@ -12,4 +12,4 @@ Duration     : 2h:21m:21s
 Inactivity   : 0h:00m:00s
 VLAN Mapping : N/A                    VLAN         : none
 Audt Sess ID : ac1063fe003fa0005e715555
-Security Grp : none
\ No newline at end of file
+Security Grp : none                   
```

Comment:
I just checked with someone who has ASA and AnyConnect and they aren't seeing this whitespace in the raw output so I think we're safe to get rid of it the `.raw` files and you can change the regex back to what it was if you'd like.

`  ^\s*Security\s+Grp\s+:\s+${SECURITY_GRP}$$ -> Record`
Should suffice

Reply:
The one output does still have spaces in it. Not sure if that is a slack thing or not. Checked and the change to use a `\s*` for greedy capture should account for both cases (whitespace and no whitespace). 

Updated the whitespace to have a mix of one with and 2 without in the single test.